python/CMakeLists.txt

```diff
@@ -50,29 +50,5 @@ endif()
 
 add_sanitizers(_scipp)
 
-set(PY_FILES
-    scipp/src/__init__.py
-    scipp/src/extend_units.py
-    scipp/src/object_list.py
-    scipp/src/show.py
-    scipp/src/table.py
-    scipp/src/compat/mantid.py
-    scipp/src/neutron/__init__.py
-    scipp/src/neutron/exceptions.py
-    scipp/src/neutron/load.py
-    scipp/src/neutron/diffraction/__init__.py
-    scipp/src/neutron/diffraction/load.py
-    scipp/src/plot/__init__.py
-    scipp/src/plot/config.py
-    scipp/src/plot/dispatch.py
-    scipp/src/plot/plot_1d.py
-    scipp/src/plot/plot_2d.py
-    scipp/src/plot/plot_3d.py
-    scipp/src/plot/plot_collapse.py
-    scipp/src/plot/plot_matplotlib.py
-    scipp/src/plot/plot.py
-    scipp/src/plot/tools.py
-)
```

Comment:
Why did you remove them?

Reply:
It seems that they are not used at all?
CMake apparently finds and copies all the files in the subdirectories.
See discussion on Slack (scipp channel)